source/migration/BuildTasks/Xamarin.AndroidX.Migration.nuspec

```diff
@@ -27,6 +27,9 @@ This package provides a set of tools and MSBuild tasks to aid in the migration f
       <group targetFramework="monoandroid90">
         <dependency id="Xamarin.AndroidX.MultiDex" version="$multidex$" include="All" />
       </group>
+      <group targetFramework="net6.0-android30.0">
```

Comment:
Should this be 31 now?

Reply:
I guess so. PR was opened on 2021-08-03, so that might be the reason.

I will fix that together with merge conflicts.